ui/src/test/java/pages/MainPage.java

```diff
@@ -2,13 +2,18 @@
 
 import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
 
 import static com.codeborne.selenide.Selenide.$;
 
+//TODO refactor extend main page, move not common things in separate classes
 public class MainPage {
 
     SelenideElement locationButton = $(".location-selector__button");
     SelenideElement locationsList = $(".location-selector__list");
+    SelenideElement LocationGlobalENButton = $(By.xpath(".//ul[@class='location-selector__list']//a[text()='Global ']"));
```

Comment:
$x(".//ul[@class='location-selector__list']//a[text()='Global ']") - $x means find by xpath in Selenide